web/config/proxy.ts

```diff
@@ -14,11 +14,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const serveUrlMap = {
```

Comment:
serverURLXX?

Reply:
> serverURLXX?

refer to https://pro.ant.design/blog/proxy-cn#%E9%AB%98%E7%BA%A7%E7%94%A8%E6%B3%95

Reply:
ok